apps/meteor/app/slashcommands-topic/server/topic.ts

```diff
@@ -1,21 +1,22 @@
 import { Meteor } from 'meteor/meteor';
-import type { IMessage } from '@rocket.chat/core-typings';
 
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { hasPermission } from '../../authorization/server/functions/hasPermission';
 
-function Topic(_command: 'topic', params: string, item: IMessage): void {
-	if (Meteor.isServer && hasPermission(Meteor.userId() as string, 'edit-room', item.rid)) {
-		Meteor.call('saveRoomSettings', item.rid, 'roomTopic', params, (err: Meteor.Error) => {
-			if (err) {
-				throw err;
-			}
-		});
-	}
-}
-
-slashCommands.add('topic', Topic, {
-	description: 'Slash_Topic_Description',
-	params: 'Slash_Topic_Params',
-	permission: 'edit-room',
-});
+slashCommands.add(
+	'topic',
+	function Topic(_command: 'topic', params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.